### ISSUE TITLE
Force stop doesn't work

### ISSUE THREAD

**Issue by sbmrvv** [ansible-semaphore/semaphore#1737]
Hi,

I installed semaphore with postgresql from [docs](https://docs.ansible-semaphore.com/administration-guide/installation#docker)

I tested my playbook on my local server. There were some changes in my playbook  that I had to push to my git repo before start the task and I forgot about it and then ran the semaphore task. When I run the task, I remembered that I have changes and decided to stop the task from running. I first click "Stop" and then "Force Stop", but even if I clicked "Force Stop", the process did not stop. It successfully completed my task. But the problem is it continued to run the task. I think it's a bag. 

![semaphore_stop_during_task](https://github.com/ansible-semaphore/semaphore/assets/149612812/cc7ee449-2450-4ed4-a9d6-cfdb9f6fc5a5)

![semaphore_task_completed](https://github.com/ansible-semaphore/semaphore/assets/149612812/6ade0d74-2c86-4d83-9d61-7bc1ce8155f6)

![semaphore_status](https://github.com/ansible-semaphore/semaphore/assets/149612812/f7819219-2df2-4a6b-b0c5-0f26e98ebbdb)

I don't know which version I have. But I installed a semaphore about 4 months ago. It could be one of those docker images above.
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/1c5dbe4b-9758-4ca2-b0c1-61afa17f1bcc)

**Comment by fiftin** [comment]
@sbmrvv could you upgrade to latest beta? But make backup of database and `config.json`. I think the issue was fixed.

**Comment by sbmrvv** [comment]
@fiftin I'm afraid the problem has not yet been resolved. Tried to stop twice, it ignores stop command)

![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/a9cdecc3-07ba-4de8-98c1-6e191d1eae65)
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/85c19569-b98d-44f6-835a-54084ccb9461)
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/a1441161-0818-4647-bdba-db3d73f55008)

**Comment by sbmrvv** [comment]
Upgraded to the latest version
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/def2c238-0bf4-4e9b-8647-a757eb21ea10)

**Comment by sbmrvv** [comment]
@fiftin I think you should try it to clear up the bug.

**Comment by tboerger** [comment]
Latest is not the beta release @fiftin was refering to :)

**Comment by sbmrvv** [comment]
I wanted to provide a final update. Interestingly, the problem resolved itself without any direct actions on my part. Since I reported this, I attempted to stop a task again and it behaved as expected, providing a `failed` status and halting the playbook execution properly.
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/d7118c06-66ee-4fd5-9ec7-c4dbb9ca1b1d)
![image](https://github.com/ansible-semaphore/semaphore/assets/149612812/39ebb30a-99fd-455a-b9a2-f61652c8e865)

It appears to have been a transient issue, possibly related to specific conditions or temporary glitches in the system at the time. I haven't been able to reproduce the original problem since then.

Given this, I am closing this issue as it no longer represents a problem and to help keep the project's issue tracker more organized. Should anyone encounter something similar, it might be worthwhile to check if the issue persists after a period or under different conditions.

Thank you to everyone who took the time to look into this or offered advice. Your support is greatly appreciated!
